mujoco_ros/include/mujoco_ros/plugin_utils.h

```diff
@@ -51,7 +51,7 @@ class MujocoPlugin
 	void init(const XmlRpc::XmlRpcValue &config, ros::NodeHandlePtr nh, MujocoEnvPtr env_ptr)
```

Comment:
ros::NodeHandle doesn't need to be a pointer. It is a slim datastructure anyway.

Reply:
Keeping in mind that we might have a lot of plugins loaded in certain scenarios, does it make sense to switch to ros::NodeHandle?

Reply:
I guess you are asking whether or not to change public API?
I think, it's still time to do so ;-) If you wait longer, it become even more difficult.

Reply:
The API is changed anyway because of several switches from shared pointers to raw pointers in the callbacks, so now would be the perfect time for such a change.

My question is if there is a benefit in copying the NodeHandle for each plugin rather than using shared pointers.